SalesforceCommerceCloud/pwa-kit
# @W-15427673 Cover all allowed static assets

We need to have all allowed file types as part of MRT reference app so we are able to later run e2e tests on the deployed apps.

# Description

There is no new dependency.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added new files under static assets for the mrt reference app
 
# How to Test-Drive This PR

- Build the mrt-reference app, deploy it to an instance of Cloud, files should be accessible (with correct Content-type headers) through the bundle path.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
